Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -4561,6 +4561,7 @@ private void NightCycle()
                 if (PossibleRoles.Any(isNotInGame))
                 {
                     var roleToSee = PossibleRoles.FirstOrDefault(isNotInGame);
+                    if (roleToSee == IRole.ApprenticeSeer && !Players.Any(x => x.PlayerRole == IRole.Seer && !x.IsDead)) roleToSee = IRole.ApprenticeSeer;  //replace Seer by AppSeer if they are about to transform
```

Comment:
So if the role equals appseer and there is no seer you set the role to appseer? That makes no sense. You probably want to replace _AppSeer_ by _Seer_ if they are about to transform?

Reply:
Umm, right. BUT no, I want to replace seer by AppSeer if they're about to transform, since it's about roles that aren't in the game